fixtures/reconciler/NoopRenderer.js

```diff
@@ -0,0 +1,280 @@
+/**
```

Comment:
Can we just inline this file into the fixture index.js? Otherwise it'll keep popping up in the autocomplete when we're looking for the real one.